src/test/kotlin/org/jetbrains/research/transformations/SetupParser.kt

```diff
@@ -0,0 +1,50 @@
+package org.jetbrains.research.transformations
+
+import java.io.File
+import java.util.logging.Logger
+
+object SetupParser {
+    val LOG = Logger.getLogger(this.javaClass.name)
+    private const val TARGET_PATH = "/tmp/pythonparser"
+
+    fun putParserToTmp() {
+        //copy parser file into /tmp directory, required by Gumtree
+        try {
+            LOG.info("Putting parser into $TARGET_PATH")
+
+            val pythonparserFile = File(javaClass.getResource("./parser/pythonparser.py").path)
+            val targetFile = File(TARGET_PATH)
+
+            pythonparserFile.copyTo(targetFile)
+            makeFileExecutable()
+        } catch (e: FileAlreadyExistsException) {
+            LOG.info("Parser file is already in $TARGET_PATH")
+        }
+
+    }
+
+    fun makeFileExecutable() {
```

Comment:
It's better to pass a file as an argument (because without knowing function implementation it's unclear which file you make executable) or call the function "makeParserFileExecutable" (I prefer the former  because, in that case, your code will be more **reusable** )